README.md

```diff
@@ -69,3 +69,36 @@ The crate has one optional feature: `static-openssl`.
 When the feature is enabled, `openssl` is linked statically against a locally compiled OpenSSL.
 This can be used to create a binary with a minimal set of runtime dependencies,
 and it can make compilation easier on systems with no recent version of OpenSSL readily available.
+
+On windows, this has been reported to work.
+```
+choco install openssl
+set OPENSSL_DIR=C:\Program Files\OpenSSL-Win64
```

Comment:
```suggestion

For more information on how to build with a locally installed version of OpenSSL see:
https://docs.rs/openssl/latest/openssl/#building
```

I don't really want to advertise any specific third party package manager for Windows. I don't know enough about them to promote any specific one.

Reply:
Idk maybe neither are worth having.

Reply:
I'm okay with not including it. But the extra link might be helpful.